zmsticketprinter/src/Zmsticketprinter/Index.php

```diff
@@ -40,6 +40,39 @@ public function readResponse(
         $department = \App::$http->readGetResult('/scope/'. $scope->id . '/department/')->getEntity();
         $organisation = $ticketprinterHelper->getOrganisation();
 
+
+        /*
+         *Check whether at least one button is not active (no opening hours stored or location deactivated)
+         *the value will be transferd to the template.
+        */
+        $hasDisabledButton = false;
+        foreach ($ticketprinter->buttons as $button) {
+            if (!isset($button['enabled']) || $button['enabled'] != 1) {
+                $hasDisabledButton = true;
+                break;
+            }
+        }
+
+//        incase we want the custom template, then the close message maybe should not be in the default template
+//        if ($hasDisabledButton) {
```

Comment:
If you no longer need it then delete it. If you need it again it's in the commit history.